sdk/appconfiguration/azure-spring-cloud-starter-appconfiguration-config/README.md

```diff
@@ -287,23 +287,17 @@ public class MyClient implements ConfigurationClientBuilderSetup, SecretClientBu
 ```
 
 ## Troubleshooting
-### Enable client logging
-Azure SDKs for Java offers a consistent logging story to help aid in troubleshooting application errors and expedite their resolution. The logs produced will capture the flow of an application before reaching the terminal state to help locate the root issue. View the [logging][logging] wiki for guidance about enabling logging.
+### Logging setting
+Please refer to [Spring logging document][Spring logging document] to get more information about logging.
```

Comment:
Change `[Spring logging document][Spring logging document]` to `[Spring logging document]`